src/test/fuzz/netaddress.cpp

```diff
@@ -9,7 +9,13 @@
 
 #include <cassert>
 #include <cstdint>
+
+#ifdef WIN32
+#include <winsock2.h>
+#else
 #include <netinet/in.h>
+#endif
```

Comment:
I don't really like changing code in a build change. Maybe exclude the failing ci config via `--disable-fuzz` for now?